build/windows/win_x86.nsi

```diff
@@ -90,8 +91,8 @@ Function .onInit
                   Abort
       InstallPython:
```

Comment:
Can we test for existing python and install only if not installed?
See: [discussion](https://find-error.com/questions/65721338/installer-cannot-find-python-in-path-environment-variable-directly-after-instal)
... possibly add installed python to the %path% 

Possibly use [PYNSIS](https://pypi.org/project/pynsist/) that comes with bundled python
but has different file format. So whole NSI file would need to be reproduced.